src/app/search/search-filters.ts

```diff
@@ -1183,6 +1188,35 @@ function searchFilters(
       hasLight(item: DimItem) {
         return item.primStat && statHashes.has(item.primStat.statHash);
       },
+      curated(item: D2Item) {
+        if (!item) {
+          return false;
+        }
+
+        const socketLength = Number(
+          item.sockets && item.sockets.sockets && item.sockets.sockets.length - 2 // ignore trackers
```

Comment:
It looks like the plug has a `plugCategoryHash` of `2947756142` for trackers; just ignoring those might be more durable?